### ISSUE TITLE
Icons in Nearby list showing up when phone is set to English but not when set to Czech 

### ISSUE THREAD

**Issue by VojtechDostal** [commons-app/apps-android-commons#1852]
I noticed that icons in the Nearby list do not work when I set my phone language to Czech. Steps to reproduce: set language to Czech, reopen app, load Nearby list.

**Comment by VojtechDostal** [comment]
![screenshot_20180823-094139](https://user-images.githubusercontent.com/21329813/44512103-cb040900-a6b9-11e8-9ca7-51802612849e.jpg)
![screenshot_20180823-094226](https://user-images.githubusercontent.com/21329813/44512115-d1928080-a6b9-11e8-9aa9-9a632295c038.jpg)

**Comment by VojtechDostal** [comment]
FYI I dont remember these icons ever working in Czech. I was actually surprised to find them in English version :-)

**Comment by misaochan** [comment]
Thanks for catching this! There is probably an issue with the mapping. AFAIK, we are currently mapping icons based on this query, do you see a reason why it might not work with non-English languages?

```
     # Get the class label in the preferred language of the user, or any other language if no label is available in that language.
     OPTIONAL {
       ?item p:P31/ps:P31 ?classId.
       OPTIONAL {?classId rdfs:label ?class_label_preferred_language. FILTER (lang(?class_label_preferred_language) = "${LANG}")}
       OPTIONAL {?classId rdfs:label ?class_label_any_language}
```

**Comment by nicolas-raoul** [comment]
The problem is at https://github.com/commons-app/apps-android-commons/blob/f1f41e59f7b513d071797bd0fd0eedc1df1dcbf9/app/src/main/java/fr/free/nrw/commons/nearby/Place.java#L122

We should get the QID of the class, and match against this QID. For instance river is Q4022.

**Comment by Rulino** [comment]
I will take this issue, looks it's a general problem, acts the same for me in Slovak localization

**Comment by maskaravivek** [comment]
Sure @Rulino please take this up. :)

**Comment by domdomegg** [comment]
Fixed in #1947
